utilities/windows/codesign.sh

```diff
@@ -72,27 +72,52 @@ SERVERS=(
 NUM_RETRIES=3
 
 function do_codesign() {
+    set -x
     for retry in $(seq 1 ${NUM_RETRIES}); do
         for SERVER in ${SERVERS[@]}; do
             # Note that we're using SHA1 signing here, because that's what our certificate supports.
             # In the future, we may be able to upgrade to SHA256.
-            if signtool sign /debug /fd SHA1 /f "${CERT_PATH}" /p "${CERT_PASSWORD}" /t "${SERVER}" "$1"; then
+            if signtool sign /debug /fd certHash /f "${CERT_PATH}" /p "${CERT_PASSWORD}" /t "${SERVER}" "$1"; then
```

Comment:
Try windows filepaths for $1? Support for unix path separators is inconsistent in windows utilities

Reply:
We look exactly like https://gist.github.com/brimston3/3839a1e112fd207152569bfc99e0d7ef

Is is possibly better to just specify `SHA1` explicitly instead of making it guess that from `/f`?

Reply:
I had it as `SHA1` before, that didn't work either.  The thing that really confuses me is that it works when invoked from `inno`, but not here.